tests/ui/job-view/revisions_test.jsx

```diff
@@ -1,191 +1,195 @@
+        revisionCount={push.revision_count}
+      />,
     );
 
-    expect(wrapper.find(Revision)).toHaveLength(mockData.push.revision_count);
+    expect(getAllByTestId('revision')).toHaveLength(push.revision_count);
   });
 
   test('renders an "...and more" link if the revision count is higher than the max display count of 20', () => {
-    mockData.push.revision_count = 21;
+    push.revision_count = 21;
 
-    const wrapper = mount(
-      <RevisionList repo={mockData.repo} push={mockData.push} />,
+    const { getByText } = render(
+      <RevisionList
+        repo={repo}
+        revision={push.revision}
+        revisions={push.revisions}
+        revisionCount={push.revision_count}
+      />,
     );
-    expect(wrapper.find(MoreRevisionsLink)).toHaveLength(1);
+    expect(getByText('\u2026and more')).toBeInTheDocument();
   });
 });
 
 describe('Revision item component', () => {
-  let mockData;
-
-  beforeEach(() => {
-    const repo = new RepositoryModel({
-      id: 2,
-      repository_group: {
-        name: 'development',
-        description: '',
-      },
-      name: 'mozilla-inbound',
-      dvcs_type: 'hg',
-      url: 'https://hg.mozilla.org/integration/mozilla-inbound',
-      branch: null,
-      codebase: 'gecko',
-      description: '',
-      active_status: 'active',
-      performance_alerts_enabled: true,
-      pushlogURL:
-        'https://hg.mozilla.org/integration/mozilla-inbound/pushloghtml',
-    });
-    const revision = {
-      result_set_id: 151371,
-      repository_id: 2,
-      revision: '5a110ad242ead60e71d2186bae78b1fb766ad5ff',
-      author: 'André Bargull <andre.bargull@gmail.com>',
-      comments:
-        'Bug 1319926 - Part 2: Collect telemetry about deprecated String generics methods. r=jandem',
-    };
-
-    mockData = {
-      revision,
-      repo,
-    };
-  });
-
-  test('renders a linked revision', () => {
-    const wrapper = mount(
-      <Revision repo={mockData.repo} revision={mockData.revision} />,
-    );
-    const link = wrapper.find('a').first();
-    expect(link.props().href).toEqual(
-      mockData.repo.getRevisionHref(mockData.revision.revision),
-    );
-    expect(link.props().title).toEqual(
-      `Open revision ${mockData.revision.revision} on ${mockData.repo.url}`,
+  // let mockData;
+  //
+  // beforeEach(() => {
+  //   const repo = new RepositoryModel({
+  //     id: 2,
+  //     repository_group: {
+  //       name: 'development',
+  //       description: '',
+  //     },
+  //     name: 'mozilla-inbound',
+  //     dvcs_type: 'hg',
+  //     url: 'https://hg.mozilla.org/integration/mozilla-inbound',
+  //     branch: null,
+  //     codebase: 'gecko',
+  //     description: '',
+  //     active_status: 'active',
+  //     performance_alerts_enabled: true,
+  //     pushlogURL:
+  //       'https://hg.mozilla.org/integration/mozilla-inbound/pushloghtml',
+  //   });
+  //
+  //   mockData = {
+  //     revision,
+  //     repo,
+  //   };
+  // });
```

Comment:
Are you enabling this block in another pr?

Reply:
Oops, just forgot to delete it.  :)